### PR TITLE
Add configuration of Sublime Text's LSP package

### PR DESCRIPTION
**Pull request by Release-Candidate**
Add a description on how to configure Sublime Text to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,30 @@ It supports the following options:
...
+```yaml
+"clients": {
+    "c3-lsp": {
+      "enabled": true,
+      // The command line required to run the server.
+      "command": [
+        "c3-lsp",
+      ],
+
+      "selector": "source.c3",
```
Comment (pherrymason) [inline README.md:84]: there are also files with `c3l` extension. Should it be added here too?

Comment (Release-Candidate) [reply README.md:84]: No, this is set by the syntax package https://github.com/c3lang/editor-plugins/tree/main/sublime-text, that's why this package needs to be installed too. This is like VS Code's language definition.

Comment (Release-Candidate) [reply README.md:84]: But looking at https://github.com/c3lang/editor-plugins/blob/main/sublime-text/c3.sublime-syntax#L4 I only added the extensions `c3` und `c3t`. I'll add `c3l` there too, I guess?

Comment (pherrymason) [reply README.md:84]: I don't know about `c3t`, but `c3l` was needed for visual studio extension.

Comment (Release-Candidate) [reply README.md:84]: It's `c3i` (correct in VS Code's package.json) and the `c3t` is somehow a typo :).

**Comment by pherrymason** [comment]
Thank you!

**Comment by Release-Candidate** [comment]
You'e welcome, and thank you for the LSP

**Pull request merged**